opm/output/eclipse/EclipseReader.cpp

```diff
@@ -36,6 +37,23 @@
 namespace Opm {
 namespace {
 
+    std::vector<double> double_vector( const ecl_kw_type * ecl_kw ) {
+        size_t size = ecl_kw_get_size( ecl_kw );
+        std::vector<double> opm_data( size );
```

Comment:
> Would save you some manual work, and is slightly clearer (in my opinion).

I agree - tried something similar with no success; and then resorted to deep manual mode. I will copy your suggestion verbatim.